### ISSUE TITLE
Cannot sign in from Firefox on Mac

### ISSUE THREAD

**Issue by goldfirere** [google/codeworld#1319]
**Description**

When I click the "Sign In" button, nothing happens. (The button activates, turning green, but that's it.)

We're on MacOS 10.13.6 and Firefox 74.0 (64-bit).

**Comment by cdsmith** [comment]
This is often caused by popup blockers.  Can you check if you have one, and if so, whether disabling it fixes the issue?

**Comment by cdsmith** [comment]
If that's not it, then it would be helpful if you check the JavaScript console for errors.

**Comment by goldfirere** [comment]
You're right. Oddly, it's Ad Block Plus (https://adblockplus.org/) that's causing the problem. Disabling it for `code.world` fixes the problem.

Is there a way to detect this scenario and inform the user? I don't have Ad Block Plus on Chrome (as I rarely use Chrome), so I don't know if it's the change in plugin or the change in browser that's making the difference.

**Comment by cdsmith** [comment]
I don't know how to detect and deal with adblock.  This is a dup of https://github.com/google/codeworld/issues/64 and https://github.com/google/codeworld/issues/1075, so this has definitely come up before.

**Comment by cdsmith** [comment]
I'll leave this one open, since it has more information, and close that one as the dup
